src/index.html

```diff
@@ -6,9 +6,96 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img
+          class= "logo"
+          src="/images/logo.png"
+          alt="logo"
+        >
+      </a>
+
+      <nav>
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a
+              href="#Apple"
+              class="nav-link is-active"
+            >
+              Apple
+            </a>
+          </li>
```

Comment:
following BEM recommendations
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a
              href="#Apple"
              class="nav__link is-active"
            >
              Apple
            </a>
          </li>
```